api/v1beta1/manila_types.go

```diff
@@ -90,6 +90,10 @@ type ManilaSpec struct {
 	// NodeSelector here acts as a default value and can be overridden by service
 	// specific NodeSelector Settings.
 	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	// DBPurge parameters -
+	DBPurgeSpec DBPurge `json:"dbPurge,omitempty"`
```

Comment:
Similar to my comment below, I don't think the field's name needs to include the "Spec" suffix, because "Spec" is already part the structure's name. I prefer:

```
	DBPurge DBPurge `json:"dbPurge,omitempty"`
```
The code in cronjob.go would refer to things like "instance.Spec.DBPurge.Age"

Reply:
+1 to remove the extra "Spec" suffix